Features/Context/Override/CommonActions.php

```diff
@@ -1,79 +0,0 @@
-<?php
-
-/**
- * File containing Overriden Functions for context PlatformUI.
- *
- * @copyright Copyright (C) eZ Systems AS. All rights reserved.
- * @license For full copyright and license information view LICENSE file distributed with this source code.
- * @version //autogentag//
- */
-namespace EzSystems\PlatformUIBundle\Features\Context\Override;
-
-trait CommonActions
```

Comment:
Don't remove, deprecate (at least phpdoc, but I'd also add `trigger_error` calls. Example: `trigger_error(E_USER_DEPRECATED, "The common actions trait is deprecated, use XXX instead")`.